airbyte-integrations/connectors/source-opsgenie/integration_tests/acceptance.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+import pytest
+
+pytest_plugins = ("source_acceptance_test.plugin",)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def connector_setup():
+    """This fixture is a placeholder for external resources that acceptance test might require."""
+    # TODO: setup test dependencies if needed. otherwise remove the TODO comments
```

Comment:
Drop all the `TODO`s in the code. Not sure if you have run the formatter on this code, can you confirm?